test/funnels/automation/StableDepositorUniV3.t.sol

```diff
@@ -160,30 +170,33 @@ contract StableDepositorUniV3Test is DssTest {
 
         vm.warp(initialTime + 180);
         vm.expectRevert("StableDepositorUniV3/too-soon");
-        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, uint128(491 * WAD), uint128(491 * 10**6));
+        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, req0, req1);
 
         vm.warp(initialTime + 360);
-        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, uint128(491 * WAD), uint128(491 * 10**6));
+        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, req0, req1);
         (num, zzz,,,,,) = stableDepositor.configs(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100);
 
         assertEq(num, initNum - 2);
         assertEq(zzz, initialTime + 360);
 
-        stableDepositor.setConfig(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, -10, 360, uint96(500 * WAD), uint96(500 * 10**6), uint96(490 * WAD), uint96(490 * 10**6));
+        (uint256 expectedAmt0, uint256 expectedAmt1) = UniV3Utils.getExpectedAmounts(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, 0, 500 * WAD, 500 * 10**6, true);
+        uint96 req0_ = uint96(expectedAmt0 * 90 / 100);
+        uint96 req1_ = uint96(expectedAmt1 * 90 / 100);
+        stableDepositor.setConfig(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, -10, 360, uint96(500 * WAD), uint96(500 * 10**6), req0_, req1_);
         (initNum,,,,,,) = stableDepositor.configs(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100);
         assertEq(initNum, -10);
 
-        vm.prank(KEEPER); stableDepositor.withdraw(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, uint128(491 * WAD), uint128(491 * 10**6));
+        vm.prank(KEEPER); stableDepositor.withdraw(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, req0_, req1_);
         (num, zzz,,,,,) = stableDepositor.configs(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100);
         assertEq(num, initNum + 1);
         assertEq(zzz, initialTime + 360);
 
         vm.warp(initialTime + 540);
         vm.expectRevert("StableDepositorUniV3/too-soon");
-        vm.prank(KEEPER); stableDepositor.withdraw(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, uint128(491 * WAD), uint128(491 * 10**6));
+        vm.prank(KEEPER); stableDepositor.withdraw(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, req0_, req1_);
```

Comment:
Shouldn't in theory `req0_` and `req1_` be recalculated after a successful withdraw?

Reply:
No I think you'd need a change of price for these values to change, which would only happen after a swap.

Reply:
Ok yeah I think you are right, depositing/withdrawing shouldn't change the ratio between desired and min amount.